### DISCUSSION TITLE
Git LFS bandwith reset

### DISCUSSION THREAD

**Discussion by servoz** [community/community#141524] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

We had too many large files managed by Git LFS in our repository.
So we deleted all these files from the history and we deleted our project (populse) from GitHub in order to completely clean up the data on the Git LFS server, then I reinstalled the project.
Since then, we've very little Git LFS data but the bandwidth was still blocked by the maximum quota.
So I waited for the monthly reset before I could use Git LFS again.

The scheduled reset date has taken place, as it now [says that the next one will take place in 31 days' time](https://github.com/organizations/populse/settings/billing/summary). However, this morning I couldn't retrieve the files tracked by Git LFS. Do you have an explanation for this? I thought the bandwidth would reset automatically.

Thanks.

**Comment by Adamjohansen31** [comment]
GitHub might not have fully released the bandwidth yet. [Sophie Rain](https://socialmediagirlsforum.us/sophie-rain-socialmediagirls/), one of the socialmediagirls, faced the same issue and had to contact support.
